x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/optimizer/PhysicalPlanOptimizerTests.java

```diff
@@ -1892,6 +1892,110 @@ public boolean exists(String field) {
         assertThat(Expressions.names(localSourceExec.output()), contains("languages", "min", "seen"));
     }
 
+    /**
+     * Expects
+     * intermediate plan
+     * LimitExec[500[INTEGER]]
+     * \_AggregateExec[[],[COUNT(emp_no{f}#6) AS c],FINAL,null]
+     *   \_ExchangeExec[[count{r}#16, seen{r}#17],true]
+     *     \_FragmentExec[filter=null, estimatedRowSize=0, fragment=[
+     * Aggregate[[],[COUNT(emp_no{f}#6) AS c]]
```

Comment:
Can you put this line on the same as the one above. It will fit the IDE line size limit.